src/js/components/account/login.js

```diff
@@ -0,0 +1,89 @@
+import { API } from "../../api";
+import { setMenuState } from "../../utils/menu";
+import {
+  getDiv,
+  getForm,
+  getInput,
+  getLabel,
+  getSubmitButton,
+} from "../../utils/element-factory";
+
+export class Login {
+  constructor() {
+    const $successTemplate = $(".styles .form-styles .w-form-done").clone();
+    const $failTemplate = $(".styles .form-styles .w-form-fail").clone();
+
+    const defaultBaseUrl = "https://muni-portal-backend.openup.org.za";
+    const endPoint = "/api/accounts/login/";
+    const fields = [
+      {
+        label: "login",
```

Comment:
this isn't label, this is name - is this just a quick hack to get things working?